web-app/pages/404.tsx

```diff
@@ -0,0 +1,25 @@
+export default function Example() {
```

Comment:
Please change from `Example` to something more relevant.

Reply:
done